cluster/gce/configure-vm.sh

```diff
@@ -166,7 +166,7 @@ download-or-bust() {
     for url in "${urls[@]}"; do
       local file="${url##*/}"
       rm -f "${file}"
-      if ! curl -f --ipv4 -Lo "${file}" --connect-timeout 20 --max-time 80 --retry 6 --retry-delay 10 "${url}"; then
+      if ! curl -f --ipv4 -Lo "${file}" --connect-timeout 20 --max-time 300 --retry 6 --retry-delay 10 "${url}"; then
```

Comment:
configure-vm definitely is: https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/configure-vm.sh#L21

I was thinking about this though...

1.5 GB  in 80 seconds is about 20 MB/second.    300 seconds is about 5 MB / second.  I would argue we probably shouldn't treat anything over 1MB/s as an error, but a 1500 second timeout just feels ridiculous.  The right answer is to make the download smaller (or skip it entirely), but in the meantime...